openapi.json

```diff
@@ -560,6 +562,41 @@
         "type": "object",
         "required": ["prompt_name"],
         "title": "GetByName"
+      },
+      "Model": {
+        "title": "Model",
+        "type": "object",
+        "properties": {
+          "provider": { "title": "Provider", "type": "string" },
+          "name": { "title": "Name", "type": "string" },
+          "parameters": {
+            "title": "Parameters",
+            "default": {},
+            "type": "object"
+          }
+        },
+        "required": ["provider", "name"]
```

Comment:
Why are these required? Can't they be null?

Reply:
@jzone3 don't we always need a provider and name for the model?

Reply:
What if the user doesn't want to specify any metadata at all?

Reply:
@jzone3 the user can choose not to send metadata at all or send it as null.